docs/installation.rst

```diff
@@ -73,12 +60,20 @@ raft
     `pysyncobj` module in order to use python Raft implementation as DCS
 aws
     `boto3` in order to use AWS callbacks
+all
+    all of the above (except psycopg family)
+psycopg
+    `psycopg[binary]>=3.0.0` module
```

Comment:
This line differs from the corresponding one in `README.rst`.
I guess we should adjust either of them, probably in `README.rst` given the commit message.

Reply:
The code in `setup.py` and the psycopg3 docs page also suggest the one to be fixed is `README.rst`.